test/e2e/fixture-server.js

```diff
@@ -94,15 +92,11 @@ class FixtureServer {
     });
   }
 
-  async loadState(directory) {
-    const statePath = path.resolve(__dirname, directory, 'state.json');
-
+  async loadJsonState(statePath, rawState) {
     let state;
     if (this._initialStateCache.has(statePath)) {
       state = this._initialStateCache.get(statePath);
     } else {
-      const data = await fs.readFile(statePath);
-      const rawState = JSON.parse(data.toString('utf-8'));
       state = performStateSubstitutions(rawState);
       this._initialStateCache.set(statePath, state);
```

Comment:
The caching here doesn't really make sense anymore. It's probably only resulting in increased memory usage, because we're still getting a new state object for each test. This state cache is now nothing more than a memory leak.

I think we can remove this caching logic, along with the `statePath` parameter.

Reply:
Agreed